x/launch/client/cli/query.go

```diff
@@ -35,6 +35,8 @@ func GetQueryCmd(queryRoute string) *cobra.Command {
 	cmd.AddCommand(CmdShowRequest())
 	cmd.AddCommand(CmdListRequest())
 	cmd.AddCommand(CmdQueryParams())
+	cmd.AddCommand(CmdRequestSelection())
```

Comment:
`AddCommand` is a variadic function. Can you put all `AddCommand` calls in the same one?
```go
	cmd.AddCommand(
		CmdShowChain(),
		CmdListChain(),
		CmdShowGenesisValidator(),
		CmdListGenesisValidator(),
		CmdShowVestingAccount(),
		CmdListVestingAccount(),
		CmdShowGenesisAccount(),
		CmdListGenesisAccount(),
		CmdShowRequest(),
		CmdListRequest(),
		CmdQueryParams(),
		CmdRequestSelection(),
	)
```